cmd/kubeadm/app/constants/constants.go

```diff
@@ -31,6 +31,9 @@ const (
 	APIServerKubeletClientCertName           = "apiserver-kubelet-client.crt"
 	APIServerKubeletClientKeyName            = "apiserver-kubelet-client.key"
 
+	ServiceAccountKeyBaseName = "sa"
+	ServiceAccountKeyName     = "sa.key"
```

Comment:
I think I might be missing something. The only thing required is a private key correct? What is the public portion?